src/app/public/modules/action-hub/action-hub.component.ts

```diff
@@ -0,0 +1,106 @@
+import {
+  Component,
+  Input,
+  OnDestroy
+} from '@angular/core';
+
+import {
+  Observable,
+  Subject
+} from 'rxjs';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  SkyLink
+} from '../link-list/types/link';
+
+import {
+  NeedsAttention
+} from '../needs-attention/types/needs-attention';
+
+import {
+  Configuration
+} from './types/configuration';
+
+@Component({
+  selector: 'sky-action-hub-buttons',
+  template: `<ng-content></ng-content>`
+})
+export class SkyActionHubButtonsComponent {}
+
+@Component({
+  selector: 'sky-action-hub-content',
+  template: `<ng-content></ng-content>`
+})
+export class SkyActionHubContentComponent {}
+
+@Component({
+  selector: 'sky-action-hub',
+  templateUrl: './action-hub.component.html'
+})
+export class SkyActionHubComponent implements OnDestroy {
+  @Input()
+  public set config(value: Configuration | Observable<Configuration>) {
```

Comment:
What was the logic behind allowing for an observable here? Change detection would pick up any changes to the value without the observable so I'm not clear on why this is needed.